src/main/java/fi/aalto/cs/apluscourses/ui/ideactivities/ComponentDatabase.java

```diff
@@ -33,10 +38,44 @@
   }
 
   public static @Nullable Component getProgressButton() {
-    return ComponentLocator.getComponentsByClass("JButton")
-        .stream().filter(c -> c instanceof JButton)
-        .filter(c -> ComponentLocator.hasActionOfClass((JButton) c, "ProgressAction"))
-        .findFirst().orElse(null);
+    return null; // Changed in #654
```

Comment:
So, is this a line that will be overwritten by #654 ?

Otherwise, changes in this file look very fine!

Reply:
yes